lib/iris/fileformats/grib/_grib_cf_map.py

```diff
@@ -16,6 +16,12 @@
 # along with Iris.  If not, see <http://www.gnu.org/licenses/>.
 #
 # DO NOT EDIT: AUTO-GENERATED
+# Created on 16 February 11 12:02 from 
```

Comment:
Thanks for adding this @marqh :+1: :grin: ... just one small extra favour, presumably `16 February 11` => 11th of February 2016? Could you change the date format to be less ambiguous?